ftw/publisher/sender/tests/helpers.py

```diff
@@ -0,0 +1,37 @@
+from Acquisition import aq_base
+from ftw.publisher.sender.utils import IS_AT_LEAST_PLONE_5_1
+from plone import api
+from zope.component import getUtility
+from zope.intid import IIntIds
+import transaction
+
+
+def set_related_items(obj, *args, **kwargs):
+    related_items = list(args)
+
+    if IS_AT_LEAST_PLONE_5_1 or kwargs.get('force_relation_values', False):
+        from z3c.relationfield import RelationValue
+        intids = getUtility(IIntIds)
+
+        relations = [
+            RelationValue(intids.getId(aq_base(item)))
+            for item in related_items
+        ]
+
+        obj.relatedItems = relations
+
+        from zope.event import notify
+        from zope.lifecycleevent import ObjectModifiedEvent
```

Comment:
It's save to move these imports to the top of the file.

Reply:
I have moved the imports to the top of the module.